operator/api/v1beta1/lokistack_webhook.go

```diff
@@ -23,12 +28,17 @@ var _ webhook.Validator = &LokiStack{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (s *LokiStack) ValidateCreate() error {
-	return s.validate()
+	return s.validate(&LokiStack{})
 }
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
-func (s *LokiStack) ValidateUpdate(_ runtime.Object) error {
-	return s.validate()
+func (s *LokiStack) ValidateUpdate(old runtime.Object) error {
+	oldStack, ok := old.(*LokiStack)
+	if !ok {
+		t := reflect.TypeOf(old).String()
```

Comment:
Have we used `reflect` for this purpose before? If not, should we use `fmt.Sprintf("%t", old)` instead? This is just a question of preference, I think I would want to avoid directly referencing `reflect` and have used `Sprintf`/`Errorf` for this purpose in the past. Feel free to ignore :slightly_smiling_face:

Reply:
We use `reflect` for the same purpose in `mutate.go`. `Errorf` causes a lint error.